cli/script/command-executor.ts

```diff
@@ -875,6 +877,12 @@ function getReactNativeProjectAppVersion(command: cli.IReleaseReactCommand, proj
                 throw new Error("The specified plist file doesn't exist. Please check that the provided path is correct.");
             }
         } else {
+            // Allow the plist prefix to be specified with or without a trailing
+            // seperator character, but prescribe the use of a "-" when omitted.
```

Comment:
`separator`

(not to be rude, but i see you misspell `separate`,`separated`, etc a lot, so you may want to make a mental note that it's `separ-` and not `seper-`)